ckanext/fairdatapoint/tests/test_fair_data_point.py

```diff
@@ -70,13 +70,13 @@ def test_fdp_get_graph_pass_empty(self, mocker):
         assert fdp_get_data.call_count == 1
         assert to_isomorphic(actual) == to_isomorphic(Graph())
 
-    @pytest.mark.xfail(raises=ValueError)
+    # @pytest.mark.xpassed(raises=ValueError)
```

Comment:
Are you sure you want to keep this line commented?

Reply:
sorry, I forgot to delete it